### ISSUE TITLE
Issue running test suite on fresh clone

### ISSUE THREAD

**Issue by growlsworth** [eventualbuddha/decaffeinate#106]
I cloned your repository and tried to run the test suite. Running of sha aa253af2c13c58f9a108d2a834beb0383338ef34. Any advice? Thanks. I'm hoping to play around to see if I can hack some cjsx support. I saw your other issues and I know it will be difficult to get full cjsx support, but I'd be happy with pretty much any hacked support, even if it's never pulled into main line. I already have other hacks that I'm using that "improve" (in my opinion) the decaffinate output such as automatically running `eslint --fix` on the output, switching `var` to `const`, changing `require` to `import`. Similarly to the cjsx support, I don't think these changes would be merged into main. Anyway that's probably irrelevant, here's the issue running the test suite on a fresh clone:

```
...
The 'createClass Babel helper is used more than once in your code. It's strongly recommended that you use the "external-helpers-2" plugin or the "es2015-rollup" preset. See https://github.com/rollup/rollup-plugin-babel#configuring-babel for more information
The 'classCallCheck Babel helper is used more than once in your code. It's strongly recommended that you use the "external-helpers-2" plugin or the "es2015-rollup" preset. See https://github.com/rollup/rollup-plugin-babel#configuring-babel for more information
Treating 'assert' as external dependency
Treating 'coffee-script-redux' as external dependency
Treating 'detect-indent' as external dependency
The 'slicedToArray Babel helper is used more than once in your code. It's strongly recommended that you use the "external-helpers-2" plugin or the "es2015-rollup" preset. See https://github.com/rollup/rollup-plugin-babel#configuring-babel for more information
Treating 'magic-string' as external dependency
Treating 'assert' as external dependency
Treating 'fs' as external dependency
Treating 'path' as external dependency
Treating 'repeating' as external dependency

> decaffeinate@1.43.1 test /work/amplii/decaffeinate
> mocha build/test-bundle.js

/work/decaffeinate/build/test-bundle.js:164
  babelHelpers_createClass(Scope, [{
  ^

ReferenceError: babelHelpers_createClass is not defined
    at /work/decaffeinate/build/test-bundle.js:164:3
...
```

running node `v5.3.0` and npm `3.3.12`

**Comment by eventualbuddha** [comment]
Thanks for the report! I'll take a look at this tomorrow. As for your "hacks", some of those are probably useful for others. Have you seen my other project [esnext](https://github.com/esnext/esnext)? It does some of the things you described like switching `var` to `let` or `const` as appropriate and changing `require` to `import` and `module.exports =` to `export default` etc.

**Comment by growlsworth** [comment]
Oh esnext looks awesome. Thanks. This is way better. I'm currently forcing `const`s and then manually changing the `let`s when appropriate. The one that esnext might not do is change `var foo = function` to `function foo`. It's not what most js programmers would do, but decaffinate defaults to. I'll look into adding esnext to my script. Thanks for sharing.

**Comment by eventualbuddha** [comment]
This should be fixed by 93ed0837072c2b72ea6491aa4117e4ce498eb821, which is part of v1.44.0.

As for the `var foo = function` vs. `function foo`, I kept the same behavior as CoffeeScript because of fear of potential incompatibilities resulting from making that change. It might make sense as an option, though.

**Comment by growlsworth** [comment]
Test suite success, thanks.
